lib/OpenQA/WebAPI/Controller/API/V1/JobSettings.pm

```diff
@@ -4,6 +4,33 @@
 package OpenQA::WebAPI::Controller::API::V1::JobSettings;
 use Mojo::Base 'Mojolicious::Controller', -signatures;
 
+=over 4
+
+=item jobs
+
+Filters jobs based on a single setting key/value pair.
+
+  openqa-cli api --osd job_settings/jobs key="*_TEST_ISSUES" list_value=39911
+  openqa-cli api --osd job_settings/jobs key="*" list_value=39911
+
+=item C<key>
+
+The setting key to filter by. It accepts a string or a glob of a job variable.
+Some variables are not stored and they can not be used.
+
+=item C<list_value>
+
+The value to match for the given key. This can be a string without special characters.
+
+=back
+
+=head3 Returns
```

Comment:
This is introducing a new header, basically separating the previous items from the content after the header.
you probably wanted an `item` here as well...